serverless/serverless
# Enables service installation with a name

## What did you implement:

This PR enables to rename the service when installing from GitHub, for example `serverless install --name new-service-name --url https://github.com/SC5/serverless-messenger-boilerplate` will install `serverless-messenger-boilerplate` to folder `new-service-name` and change `service` value in `serverless.yml` to `new-service-name`.

Closes https://github.com/serverless/serverless/issues/2626

## How did you implement it:

Added new parameter `name` to install class and renameService helper function for renaming service in yaml file.

## How can we verify it:

`serverless install --name new-service-name --url https://github.com/SC5/serverless-messenger-boilerplate` should install service to `new-service-name` folder and change service name to `new-service-name`.

## Todos:



***Is this ready for review?:*** YES
